sdk/python/feast/client.py

```diff
@@ -993,6 +1004,32 @@ def _get_grpc_metadata(self):
         return ()
 
 
+def _infer_entity_rows(
+    entities: List[Dict[str, Any]]
+) -> List[GetOnlineFeaturesRequest.EntityRow]:
+    """
+    Builds a list of EntityRow protos from Python native type format passed by user.
+
+    Args:
+        entities: List of Dict[str, Union[bool,bytes,float,int,str,List[bool,bytes,float,int,str]]]]
+            where each key represents the entity name and value is feast.types.Value in Python native form.
+
+    Returns:
+        A list of EntityRow protos parsed from args.
+    """
+    entity_row_list = []
+
+    for entity in entities:
+        for key, value in entity.items():
+            # Infer the specific type for this row
+            current_dtype = python_type_to_feast_value_type(name=key, value=value)
```

Comment:
Can we calculate the current_dtype once instead of every time? It should be consistent for a column. If the casting fails then we can just fail.

Reply:
Using this loop currently to check if mixed types are present.